File: task-03/ruby/Sub-task 3.rb
print "Enter the number of rows: "
rows = gets.to_i

for i in 0...(rows) do
  for j in 0...(rows-i-1)
    print " "
  end
  for j in 0..i
    print "* "
  end
  puts
end

for i in 0...(rows-1) do
  for j in 0..i
    print " "
  end
  for j in 0...(rows-i-1)
    print "* "
  end
  puts
end

File: task-03/elixir/Sub-Task 3.ex
IO.puts("Enter the number of rows:")
rows = IO.gets("") |> String.trim() |> String.to_integer()
for i <- 0..(rows - 2) do
  for _ <- 0..(rows - i - 1) do
    IO.write(" ")
  end
  for _ <- 0..i do
    IO.write("* ")
  end
  IO.puts("")
end

for i <- 0..(rows - 1) do
  # Print leading spaces
  for _ <- 0..i do
    IO.write(" ")
  end

  # Print asterisks
  for _ <- 0..(rows - i - 1) do
    IO.write("* ")
  end

  IO.puts("")
end
